arangod/RestServer/arangod.cpp

```diff
@@ -374,6 +374,10 @@ int main(int argc, char* argv[]) {
 
   TRI_GET_ARGV(argc, argv);
 #if _WIN32
+  // Set the codepage for the console output to UTF-8 so that unicode characters
+  // are displayed correctly.
+  SetConsoleOutputCP(CP_UTF8);
```

Comment:
We have some console-related tasks in the ShellColorsFeature::prepareConsole  - maybe move this call to that function?
As a bonus we will enable UTF-8 encodings in all other executables (e.g. arangodump/arangorestore....)